### DISCUSSION TITLE
在OpenWrt上通过Docker方式安装1Panel - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by giscus** [Muska-Ami/muska-ami-blog#72] [General]
# 在OpenWrt上通过Docker方式安装1Panel - 夏沫花火zzz🌙 (Muska_Ami)のLife

安装Docker请先在OpenWrt上安装Docker，此处不做演示 部署1Panel请将 DOCKER_INSTALL_DIR 替换为你的Docker安装路径，如我的iStoreOS默认安装在 /overlay/upper/docker 12345678910docker run -d \    --name 1panel \    --restart always \    --network

https://blog.1l1.icu/2024/04/28/zai-openwrt-shang-tong-guo-docker-fang-shi-an-zhuang-1panel/

**Comment by ltzXiaoYanMo** [comment]
这下是真的是闲的了（

**Reply by Muska-Ami** [reply]
不是，怎么就闲的了（
肯定有人需要，但是查了资料发现没有一个人写教程，我写一个怎么了😡

**Reply by ltzXiaoYanMo** [reply]
就闲就闲

**Comment by selinahair** [comment]
大佬牛逼，感谢
